File: Level 0/Python/등차수열의 특정한 항만 더하기.py
def solution(a, d, included):
    tempArr = [a]
    for i in range(len(included) - 1):
        tempArr.append(tempArr[i] + d)
    
    arr=[]
    for i in range(len(included)):
        arr.append([tempArr[i], included[i]])

    return sum([key for key, value in dict(arr).items() if value == True])

File: Level 0/Javascript/기초/등차수열의 특정한 항만 더하기.js
function solution(a, d, included) {
    const tempArr = [a];
    
    for(let i = 1; i < included.length; i++) {
        tempArr.push(tempArr[i - 1] + d);
    }
    
    return tempArr.reduce((acc, cur, idx) => included[idx] ? acc + cur : acc, 0);
}